### ISSUE TITLE
When someone disconnects and reconnects again, the volume bar dissapear

### ISSUE THREAD

**Issue by juanramirez** [apajaros/talky-volume#1]
When someone disconnects and reconnects again, the volume bar dissapear

**Comment by apajaros** [comment]
Yep. We can keep on checking instead of clearing the interval, but it'd nicer if we could detect the new connection event.

**Comment by apajaros** [comment]
I've kept the interval polling for the time being so that arriving users also get the volume control. c944cceebc41502d72acf3c8ae033f283b4ea5eb
